github/activity_events.go

```diff
@@ -28,9 +28,9 @@ func (e Event) String() string {
 	return Stringify(e)
 }
 
-// Payload returns the parsed event payload. For recognized event types,
+// ParsePayload parses the event payload. For recognized event types,
 // a value of the corresponding struct type will be returned.
-func (e *Event) Payload() (payload interface{}) {
+func (e *Event) ParsePayload() (payload interface{}, err error) {
```

Comment:
Added in the old method with depreciation warning. Is it necessary to add in some tests for the old method since it's just proxying another method?